src/Momento.Sdk/Config/Retry/FixedTimeoutRetryStrategy.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using Grpc.Core;
+using Microsoft.Extensions.Logging;
+using Momento.Sdk.Internal.Retry;
+
+namespace Momento.Sdk.Config.Retry;
+
+/// <summary>
+/// Retry failed requests up until the client's timeout has been reached.
+/// Retries are attempted at fixed intervals (retryDelayIntervalMillis) +/- jitter.
+/// Retry attempts time out after responseDataReceivedTimeoutMillis to be able to 
+/// allow multiple retries before the client's timeout is reached.
+/// </summary>
+public class FixedTimeoutRetryStrategy : IRetryStrategy
+{
+    private ILoggerFactory _loggerFactory;
```

Comment:
No need to store the logger factory. You only need it to get a scoped logger to the class in the constructor.